draft-ietf-scitt-architecture.md

```diff
@@ -373,280 +369,155 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
-Examples for Statements that can improve trustworthy assessments of Transparency Services are RATS Conceptual Messages, such as Evidence, Endorsements, or corresponding Attestation Results (see {{-rats-arch}}).
+Multitenant support can be enabled through the use of identifiers in the `iss` claim, for example, `ts.example` may have a distinct issuer identity for each sub domain, such as `customer1.ts.example` and `customer2.ts.example`.
 
-For example, consider a Transparency Service implemented using a set of replicas, each running within its own hardware-protected trusted execution environments (TEEs).
-Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
-This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
+### Initialization {#ts-initialization}
 
-### Configuration
-
-The Transparency Service records its configuration in the Append-Only Log using Transparent Statements with distinguished media type `application/scitt-configuration`.
-
-The registration policy for statements with the media type suffix (`+<format>` is implementation-specific.
-The implementation SHOULD document them, for example defining the Issuers authorized to register configuration Signed Statements.
-
-The Transparency Service is configured by the last Transparent Statement of this type.
-The Transparency Service MUST register a Signed Statement that defines its initial configuration before registering any other Signed Statement.
-The Transparency Service MAY register an additional Signed Statement that updates its configuration.
-
-The Transparency Service provides an endpoint that returns the Transparent Statement that describes its current configuration.
-
-The configuration `reg_info` SHOULD include a secure version number and a timestamp.
-
-The sample configuration payload uses the CDDL {{-CDDL}}
-
-~~~ cddl
-Signature_Algorithms = [ int ]
-
-Registration_Policy = {
-  * tstr => any
-}
-
-SCITT_Configuration = [
-  supported_signature_algs: Signature_Algorithms ; supported algorithms for signing Statement
-  ledger_alg: int ; type of verifiable data structure
-  service_uri: tstr; base URI of the transparency service, will be the issuer in the receipt CWT claim set
-  root_certificates: [ COSE_X509 ]
-  supported_apis: [ SCITT_Endpoint ]
-  registration_policies : Registration_Policy
-]
-~~~
+The Append-only Log is empty when the Transparency Service is initialized.
+The first entry that is added to the Append-only Log MUST be a Signed Statement including key material.
+The second set of entries are Signed Statements for additional domain-specific Registration Policy.
```

Comment:
This description of initialization is new, and seems overly prescriptive. The first entry seems recursive, and does not say what kind of key materials are involved. In my opinion (and as written before) signed statement that document registration policies can be interleaved with other records.

Reply:
The section just aggregates "policy" stuff that was spread all over the document.

Reply:
I don't see that. The ordering constraints of entries in the log, and the exclusive focus on keys, seems fresh.

Reply:
This was the outcome of IETF 118, to formalize the initialization. Please open a new PR/Issue for any proposed changes.